app/scripts/filters/util.js

```diff
@@ -348,6 +348,30 @@ angular.module('openshiftConsole')
           .replace(/^./, function(str){ return str.toUpperCase(); });
     };
   })
+  // event messages should simple, but human readable:
+  // Pulled Pod <name>
+  // Unhealthy Pod <name>
+  // Deployment created for DeploymentConfig <name>
+  // etc.
+  // This filter handles the "Reason <optional preposition>"
+  // but not the <name> as we <a> link that in templates.
+  .filter('messageForEvent', function($filter) {
```

Comment:
Another thing to think about, some of the language lacks nuance:

![screen shot 2017-08-14 at 3 01 32 pm](https://user-images.githubusercontent.com/280512/29286872-94216c4a-8101-11e7-9a0b-69e635a32acc.png)
But more verbosely means (via events page):
![screen shot 2017-08-14 at 3 01 50 pm](https://user-images.githubusercontent.com/280512/29286873-9421a840-8101-11e7-9fa5-277ada842d76.png)
Which, in simplest language means "Pulled Image for Pod xzy".